docker/swe.Dockerfile

```diff
@@ -5,6 +5,7 @@ ARG MINICONDA_URL
 # Install third party tools
 RUN apt-get update && \
     apt-get install -y bash gcc git jq wget g++ make && \
+    apt-get install -y nodejs npm && \
```

Comment:
seems like this will bloat the image as more languages are added according to the thrust of [this section](https://github.com/princeton-nlp/SWE-agent/pull/83/files#diff-404c46b336280a67ec4b902a13f28d1cb24fdc2f17f608bc586eeedfada91ca7R27). I'm no docker expert but I wonder if there's a way to parameterize this dockerfile somehow

Reply:
Some of the initial setup of the container still passes in manual install commands, so we could technically move this there if container bloat becomes an issue